api.js

```diff
@@ -1801,7 +1803,114 @@ module.exports = (function () {
       });
     });
 
+    app.get(padRoute + '/getLastEdited/:key', function (req, res) {
```

Comment:
Is it possible to return the lastEdited value immediately In the getrelevantpads request.  This allows us to save on multiple request calls later on.